omniport/omniport/static/full-calendar-semantic-ui/semantic.fullCalendar.css

```diff
@@ -0,0 +1,74 @@
+.fc-today {
+  background: #fcf8e3;
+  font-weight: bold;
+}
+
+.fc-today-button {
+  margin: 0.5em;
+  border: 1px solid rgba(34, 36, 38, .15);
+  border-radius: .28571429rem;
+  padding: .78571429em 1.5em .78571429em;
+  box-sizing: border-box;
+  background-color: #FFFFFF;
+  color: #000000;
+  font-family: Lato;
+  font-weight: bold;
+}
+
+.fc-button-group {
+  margin: 0.5em !important;
+  border-radius: .28571429rem !important;
+}
+
+.fc-prev-button, .fc-next-button, .fc-dayGridMonth-button, .fc-timeGridWeek-button, .fc-timeGridDay-button, .fc-listWeek-button {
+  padding: .78571429em 1.5em .78571429em;
+  background: #FFFFFF;
+  border: 1px solid rgba(34, 36, 38, .15);
+  box-sizing: border-box;
+  color: #000000;
+}
+
+.fc-dayGridMonth-button:focus {
+  border-end-end-radius: 0px;
+}
+
+.fc-timeGridWeek-button:focus, .fc-timeGridDay-button:focus {
+  border-radius: 0px;
+}
+
+.fc-listWeek-button:focus {
+  border-start-start-radius: 0px;
+}
+
+.fc-center h2 {
+  font-style: normal;
+  font-weight: bold;
+  font-family: Lato;
+  box-sizing: border-box;
+  color: #000000;
+}
+
+.fc-button-active {
+  color: #fff;
+  padding: .78571429em 1.5em .78571429em;
+  cursor: default;
+  background-color: #EBEBEB;
+  border: 1px solid rgba(34, 36, 38, .15);
+}
+
+.fc-button:focus {
+  outline: 0px;
+  border: 1px solid rgba(34, 36, 38, .15);
+  box-shadow: 0px 0px 0px red;
+  padding: .78571429em 1.5em .78571429em;
+  box-sizing: border-box;
+  color: #000000;
+}
+
+.fc-button-primary:focus {
+  box-shadow: 0px 0px 0px red;
+}
+
+.fc-day-number {
+  color: #3a87ad;
+}
```

Comment:
@SuyashSalampuria please add a new line here. That's all I can say :laughing:

Reply:
@SuyashSalampuria reminder lets merge this after this change, other things seem fine to me. The corresponding PR on the frontend can be merged only after this PR.

Reply:
Done @pradumangoyal